.gitignore

```diff
@@ -58,3 +58,4 @@ ghostdriver.log
 /kalite/_built.touch
 /kalite/contentload/tests/fixtures/
 *.swp
+/data/khan/assessmentitems.json.version
```

Comment:
`assessmentitems.json.version` has the KA Lite version of the software that generated this zip file.
a.k.a
"0.13"